### PR TITLE
Minor cosmetics

### PR DESCRIPTION
**Pull request by sofianehaddad**
The PR provides minor changes : 
 - Fix potential circular/recursive import in viewer
 - Provide a SVG export to drawables

### CODE REVIEW

**Comment by sofianehaddad** [comment]
Ah exact! 
No idea, except a static View.Show
Any idea?

**Comment by jschueller** [comment]
Yeah, I was thinking the same thing. Maybe 'ShowAll' to differenciate it more ?

**Comment by sofianehaddad** [comment]
ok I will add it

File: python/src/viewer.py [hunk 1]
```diff
@@ -514,6 +514,14 @@ def show(self, **kwargs):
             # mpl < 1.3.1, see https://github.com/ipython/ipython/pull/1615
             plt.show(**kwargs)
 
+    @staticmethod
+    def ShowAll(**kwargs):
+        """
+        Display all graphs.
+
```
Comment (jschueller) [inline python/src/viewer.py]: no new line here (pep257)

Comment (sofianehaddad) [inline python/src/viewer.py]: fixed

Comment (sofianehaddad) [inline python/src/viewer.py]: Done

**Comment by jschueller** [comment]
There's a Show function in __init__.py that also assumes a blocking behavior.
But I suggest we remove it ; its undocumented anyways.

**Comment by sofianehaddad** [comment]
@jschueller no the Show function does not assumes a blocking behaviour
It depends on matplotlib version + used backend

**Comment by jschueller** [comment]
if you use the default backend on an oridnary debian it's blocking.

**Comment by sofianehaddad** [comment]
@jschueller I use default backend with an ordinary debian ==> not blocking

Btw the Show method is very usefull; I think we should not remove it

**Comment by sofianehaddad** [comment]
@jschueller Show function in **__init__** is documented

    Display a graph pop-up.

    Parameters
    ----------
    graph : Graph
        the graph to display

**Comment by jschueller** [comment]
I don't think there's an rst entry

**Comment by sofianehaddad** [comment]
yep

**Comment by regislebrun** [comment]
@sofianehaddad I agree with you: the Show() command is REALLY usefull. It is a one-line version of the 3 to 4 lines viewer + the specific import. I use it mainly for debuging purpose, for production graphics Viewer is much more adapted. I confirm that the Show() command was blocking in the 1.8 version. It is no more the case with the current trunk, and it is a regression from my point of view (even if it solved other issues).

otherwise +1

**Comment by jschueller** [comment]
@sofianehaddad it's blocking for me too, matplotlib.get_backend() return TkAgg, what's yours ? and @regislebrun ?

**Comment by sofianehaddad** [comment]
@jschueller same backend but not blocking

**Comment by jschueller** [comment]
Anyways, it uses view.show which used plt.show(), to get the same behavior it should use View.ShowAll now.

**Comment by sofianehaddad** [comment]
@jschueller did not understand
you want Show using View.ShowAll instead of View.show?

**Comment by jschueller** [comment]
yes, to result in a call to plt.show as before

**Comment by sofianehaddad** [comment]
Done

**Comment by jschueller** [comment]
Thanks
Could you rebase to master ? I tried to fix the circle-ci script from a forked repo without passing secret env vars.

**Pull request merged**